pkg/api/v1/conversion.go

```diff
@@ -373,63 +372,14 @@ func Convert_api_PodStatusResult_To_v1_PodStatusResult(in *api.PodStatusResult,
 		return err
 	}
 
```

Comment:
I wonder what these functions do? Not sure about that.

Also, seems like `Convert_v1_ReplicationControllerSpec_To_api_ReplicationControllerSpec` invokes `Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec`, does this mean `Convert_v1_ReplicationControllerSpec_To_api_ReplicationControllerSpec` also can be removed?

Thanks @liggitt !